site/ko/blog/new-in-devtools-100/index.md

```diff
@@ -28,19 +28,24 @@ draft: true
 <!-- end: translation instructions -->
 
```

Comment:
Please remove the `draft:true` and translate the `description` metadata too!

Reply:
Applied